src/core/shape/vertex.js

```diff
@@ -626,8 +627,12 @@ p5.prototype.endContour = function() {
  * void main() {
  *
  *   vec4 positionVec4 = vec4(aPosition, 1.0);
+ * 
+ *   // gl_InstanceID represents a numeric value for each instance
+ *   // using gl_InstanceID allows us to move each instance separately
+ *   // here we move each instance horizontally by id * 100
  *   gl_Position = uProjectionMatrix * uModelViewMatrix * positionVec4 -
- *   vec4(vec2(float(gl_InstanceID) * 100.0, 0.0), 0.0, 0.0);
+ *     vec4(vec2(float(gl_InstanceID) * 100.0, 0.0), 0.0, 0.0);
```

Comment:
can you simplify this line to be `vec4(float(gl_InstanceID) * 100.0, 0.0, 0.0, 0.0);`. There's no need for the vec2 here.

Reply:
That looks good